terraform/electionleaflets/dns.tf

```
@@ -8,90 +8,18 @@ resource "cloudflare_zone" "main" {
 resource "cloudflare_record" "root" {
   zone_id = cloudflare_zone.main.id
   name    = "electionleaflets.org.au"
+  comment = "Used to create Redirect Rule"
+  proxied = false
```

Proxy needs to be enabled
```suggestion
  proxied = true
```